entrypoint.sh

```diff
@@ -26,4 +26,23 @@ cat > config.json << EOF
 }
 EOF
 
-./smb config.json
+cat > nginx.conf << EOF
+events {}
+http {
+  server {
+    listen  0.0.0.0:${HTTP_PORT};
+    location / {
+      proxy_set_header Host \$host;
+      proxy_pass http://127.0.0.1:8181;
+      add_header X-Proxy yes always;
+    }
+    location = / {
```

Comment:
There is a health endpoint on smb under /about/health. These requests could possibly be redirected there.